pandas/tests/test_downstream.py

```diff
@@ -55,6 +55,30 @@ def test_dask(df):
         pd.set_option("compute.use_numexpr", olduse)
 
 
+@pytest.mark.filterwarnings("ignore:.*64Index is deprecated:FutureWarning")
```

Comment:
can you use the import decorator here e.g. `@td.skip_if_no("xarray"` (and can you fix the L39 one

i also don't know why we have a special import_module here.

Reply:
This seems to be done on purpose:

https://github.com/pandas-dev/pandas/blob/4ce84de5250aafa706d4c7610fdefad0baf37acf/pandas/tests/test_downstream.py#L23-L30

Reply:
ok really this should be cleaned up cc @jbrockmendel